spec/predicates/cyclonedx.md

```diff
@@ -0,0 +1,71 @@
+# Predicate type: CycloneDX
+
+Type URI: (tentative) https://cyclonedx.org/bom
+
+Version: 1.0.0
+
+## Purpose
+
+A Software Bill of Materials type following the [CycloneDX standard].
+
+This allows representing "exportable", or "published" software artifacts,
+services, vulnerability information, and more. For a complete list of
+capabilities see [CycloneDX Capabilities].
+
+## Prerequisites
+The in-toto [attestation] framework.
+
+## Model
+This is a predicate type that fits within the larger [Attestation] framework.
+
+## Schema
+The schema of this predicate type is documented in the
+[CycloneDX Specification].
+
+### Parsing Rules
+The parsing rules for this predicate type are documented in the
+[CycloneDX Specification].
+
+### Fields
+The fields that make up this predicate type are documented in the
+[CycloneDX Specification].
+
+## Example
+```jsonc
+{
+  // Standard attestation fields:
+  "_type": "https://in-toto.io/Statement/v0.1",
+  "subject": [{ ... }],
+
+  // Predicate:
+  "predicateType": "https://cyclonedx.org/BOM",
```

Comment:
CDX schema version should be part of the `predicateType`

Reply:
+1

Reply:
As stated below, the `specVersion` field is required by the CDX spec, which is why the version is not in the `predicateType`. See: https://cyclonedx.org/docs/1.4/json/#bomFormat

Reply:
I still think it should be part of the type such that we don't have to go down to the predicate to understand the `specVersion`. I am thinking about this in terms of tooling that might process these in the future.

Reply:
That's totally fair. Probably shouldn't assume that all tools will know how to process CDX predicates.

Reply:
Apologies for my late reply here!

So would something like this work:
```
"predicateType": "https://cyclonedx.org/BOM/v1.4"
```